### PR TITLE
Automate copy analysis profiles

### PR DESCRIPTION
**Pull request by akarol**
{{pytest: -v -k 'test_host_analysis_profile_crud'}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 83ba45b021716c93b3aca40b1ba823d63689bc16:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/configure/configuration/__init__.py [hunk 1]
```diff
@@ -507,6 +507,17 @@ def delete(self):
...
 
+    def copy(self, name=None):
+        if not name:
+            name = self.name + "copy"
+        sel.force_navigate("cfg_analysis_profile", context={"analysis_profile": self})
+        tb.select('Configuration', 'Copy this selected Analysis Profile')
+        new_profile = AnalysisProfile(name=name, description=self.description, files=self.files)
+        fill(self.form, {'name': new_profile.name},
+             action=form_buttons.add)
+        flash.assert_success_message('Analysis Profile "%s" was saved' % new_profile.name)
```
Comment (mfalesni) [inline cfme/configure/configuration/__init__.py]: ~~%s~~  `{}`

**Comment by dajoRH** [comment]
Lint report for commit 2ada69fd85e9ac1730e7952db56f4dc5637173e8:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/configure/configuration/__init__.py (hunk 1 above)
Comment (akarol) [inline cfme/configure/configuration/__init__.py]: @mfalesni done!

**Pull request merged**